### ISSUE TITLE
Config ui dont close when loses focus

### ISSUE THREAD

**Issue by audoban** [psifidotos/Latte-Dock#14]

**Comment by psifidotos** [comment]
I disabled this because right clicking inside the Latte Dock was hiding the configuration window... 

Any ideas for this ?

**Comment by audoban** [comment]
But, this _focusWindow->flags().testFlag(Qt::Popup)_ condition avoid this.
```
void DockConfigView::focusOutEvent(QFocusEvent *ev)
{
    Q_UNUSED(ev);
    const auto *focusWindow = qGuiApp->focusWindow();
    
    if (focusWindow && focusWindow->flags().testFlag(Qt::Popup))
        return;
        
    hide();
}
```

**Comment by psifidotos** [comment]
this has been fixed from your commits...

we can can use e.g. **fix #14** at the header of commits in order to close issues automatically when merged to mater

**Comment by audoban** [comment]
ok thanks.
